README.md

```diff
@@ -214,17 +216,14 @@ This means that the following will not trigger a sound:
 
 Please [open an issue][issue] if you want any other text events to play unique sounds.
 
-## WAV is the only sound fileformat playable on Windows
+## WAV is the only sound file format playable on Windows
 
 HexChat officially supports Linux and Windows, and uses different mechanisms to play sound files on each platform.
 
 On Windows, HexChat calls [`PlaySound`][PlaySound], which can process some kinds of [WAV][] files.
 
 [Audacity][] can be used to [import audio][import-audio], and then [export the audio][export-audio] in one of the following formats understood by [`PlaySound`][PlaySound]:
 
-* Signed 16-bit PCM
-* Signed 16-bit PCM
-* Signed 16-bit PCM
 * Signed 16-bit PCM
 * 32-bit float
 * μ-Law

```

Comment:
u-law and a-law are both raw format audio. Consider removing, even if accurate?

Reply:
While in large part the sound data is the same,

![image](https://user-images.githubusercontent.com/51795309/67800375-27e51700-fa55-11e9-9010-815630682a70.png)

[`PlaySound`](https://docs.microsoft.com/en-us/previous-versions/dd743680(v%3Dvs.85)) does not play the file without the WAV header.

Reply:
Additionally, both algorithms compress the data as compared to standard Signed 16-bit PCM:

![image](https://user-images.githubusercontent.com/51795309/67801108-852d9800-fa56-11e9-9c79-4b3a3b3879cd.png)

Reply:
You're not wrong. but they also lack any form of appreciable header, it mostly just dumps the raw audio data onto disk... and the same in reverse. I can open literally any file on my computer as a valid u-law / a-law and it'll import correctly.

Reply:
Unfortunately, [`PlaySound`](https://docs.microsoft.com/en-us/previous-versions/dd743680(v%3Dvs.85)) requires files to be structured like [`WAVEFORMATEX`](https://docs.microsoft.com/en-us/previous-versions/dd757713(v=vs.85)) or similar.